src/sagemaker/serializers.py

```diff
@@ -192,6 +192,34 @@ def serialize(self, data):
         return json.dumps(data)
 
 
+class JSONLinesSerializer(BaseSerializer):
+    """Serialize data to a JSON Lines formatted string."""
+
+    CONTENT_TYPE = "application/jsonlines"
+
+    def serialize(self, data):
+        """Serialize data of various formats to a JSON Lines formatted string.
+
+        Args:
+            data (object): Data to be serialized. The data can be a string,
+                list of JSON serializable objects, or a file-like object.
+
+        Returns:
+            str: The data serialized as a string containing newline-separated
+                JSON values.
+        """
+        if isinstance(data, str):
+            return data
+
+        if isinstance(data, list):
```

Comment:
Should the serializer support the more generic `Iterable` from `collections.abc`?

```suggestion
        if isinstance(data, Iterable):
```

Reply:
Fixed